app/bundles/CoreBundle/Controller/CommonController.php

```diff
@@ -668,11 +668,7 @@ public function addBrowserNotification($message, $messageVars = [], $domain = 'f
             //message is already translated
             $translatedMessage = $message;
         } else {
-            if (isset($messageVars['pluralCount'])) {
-                $translatedMessage = $translator->transChoice($message, $messageVars['pluralCount'], $messageVars, $domain);
-            } else {
-                $translatedMessage = $translator->trans($message, $messageVars, $domain);
-            }
+            $translatedMessage = $translator->trans($message, $messageVars, $domain);
```

Comment:
Shouldn't this transfer the `pluralCount` into `count`? At least for the BC purposes so we could merge it into the 4.x branch

```suggestion
            if (isset($messageVars['pluralCount']) && empty($messageVars['%count%'])) {
                $messageVars['%count%'] = $messageVars['pluralCount'];
            }
            $translatedMessage = $translator->trans($message, $messageVars, $domain);
```

Reply:
I checked, there are not many instances where pluralCount is being used.
Instead we can use count directly. What do you think?

Reply:
I imagine there is a plugin that is calling the `addBrowserNotification` with the `$messageVars['pluralCount']` param and no `$messageVars['%count%']`. If so then the translation will stop working. Or won't it?

Reply:
Gotcha! Yes, it will stop. Accepting your suggestions.